### ISSUE TITLE
No documentation on "noise enhanced" T1w image

### ISSUE THREAD

**Issue by nicholst** [poldracklab/mriqc#691]
For T1w HTML reports there is a mosaic of images entitled "noise enhanced".  Searching the docs finds no reference to this (except in source code).  

Is this just an aggressively windowed image?

**Comment by oesteban** [comment]
Yes, it is just that.There is some documentation in the works (https://1387-54756129-gh.circle-artifacts.com/1/home/ubuntu/scratch/docs/interpreting.html#the-t1w-noise-enhanced-plot) but I'll leave this issue open to improve the documentation specifically for this "noise enhanced" T1w plot.

Thanks!
